lib/nexmo_developer/app/services/translator/file_translator.rb

```diff
@@ -0,0 +1,43 @@
+module Translator
+  class FileTranslator
+    attr_reader :doc_path
+
+    def initialize(doc_path)
+      @doc_path = doc_path
+    end
+
+    def translation_requests
+      doc = load_frontmatter
```

Comment:
you don't really need to call this here, on line 22 you can just call it like this

```
return frontmatter['translation_frequency'] if frontmatter['translation_frequency']
```

Reply:
But I use the `doc` in the method call later down in this method, so I do need it.

Reply:
you don't need to call it here, instead of passing it as a parameter to `#frequency_calculator`, you can just call `#frontmatter` in that method

Reply:
Got it, that makes sense. Thanks.